ome/resource_discovery.go

```diff
@@ -114,6 +213,29 @@ func (r *discoveryResource) Update(ctx context.Context, req resource.UpdateReque
 		return
 	}
 
+	if !reflect.DeepEqual(state, plan) {
```

Comment:
Since u are in the update procedure, will this condition ever be false?

Reply:
removed the conditions